cubedash/_stac.py

```diff
@@ -513,7 +513,12 @@ def _format(val: str) -> dict[str, str]:
         # default is ascending
         return {"field": val.strip(), "direction": "asc"}
 
-    arg_work = arg.split(",") if isinstance(arg, str) else arg
+    if isinstance(arg, str):
```

Comment:
This part seems to fix a type error, so that is usually good.

(I've noticed your PRs sometimes fail on the type checker, you can run mypy locally on the file before pushing with `uv run mypy --follow-imports=silent cubedash/_stac.py`, and for other ODC repositories the `--follow-imports` can be skipped.)

Reply:
My local mypy often gives different results to the one run in the CI, despite being the same version.

Reply:
That's really odd! And probably worth resolving... :-)

Reply:
@Ariana-B if `uv run mypy <..>` gives you errors that aren't seen in CI, I think there's something wrong in the mypy configuration. Can you please write down the exact command you used to run mypy, the error you got, and the repository where you got it the next time it happens for you?